editor/editor_node.cpp

```diff
@@ -3286,8 +3286,8 @@ void EditorNode::_update_file_menu_closed() {
 	file_menu->set_item_disabled(file_menu->get_item_index(FILE_OPEN_PREV), false);
 }
 
-Control *EditorNode::get_main_control() {
-	return main_control;
+Control *EditorNode::get_main_screen_control() {
```

Comment:
```suggestion
Control *EditorNode::get_main_screen() {
```

Reply:
That was on purpose. I wanted a bit more specificity for the internal method.